app/views/lab_tests/_date_filter.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="mb-6 flex justify-between items-center">
+    <%= form_with(url: lab_tests_path, method: :get, class: "inline-flex items-center") do |f| %>
+    <div class="inline-flex items-center">
+        <span class="mr-3 text-sm font-medium"><%= t('.lab_test_date_filter.from') %></span>
```

Comment:
There are 3 broken translation links here:
![image](https://github.com/user-attachments/assets/e235b948-fba1-4975-a56c-588b3955b827)

Seems that you have changed them in locals config, but not here.

Reply:
That's why there is a bugged button on the index page:
![image](https://github.com/user-attachments/assets/ea883f62-3465-4299-b559-5498a24897f4)